src/ray/protobuf/experimental/autoscaler.proto

```diff
@@ -120,6 +118,14 @@ message NodeState {
 
   // The status of the node.
   NodeStatus status = 17;
+
+  message IdleInfo {
```

Comment:
should we add IDLE in the NodeStatus as well?

Reply:
i guess we could.